deepcell_toolbox/deep_watershed.py

```diff
@@ -102,15 +102,16 @@ def deep_watershed(outputs,
 
 
 def deep_watershed_mibi(model_output,
-                        min_distance=10,
+                        radius=10,
                         maxima_threshold=0.1,
                         interior_threshold=0.2,
-                        exclude_border=False,
```

Comment:
again, let's not remove inputs. if someone updates code their codebase may crash.